### ISSUE TITLE
[Bug] In `Card info`, the date of `Due` is shown on a different format from the one of the others  

### ISSUE THREAD

**Issue by snowtimeglass** [ankidroid/Anki-Android#10637]
Thanks to developers.

###### Reproduction Steps

1. `Settings` -> `AnkiDroid - General settings` ->  `Language` -> `English`
2. Show `Card info` of a card

###### Expected Result
The dates of `Added`, `First Review`, `Latest Review` and `Due` are shown on a common format. 

###### Actual Result
The date of `Due` is shown on a different format from the one of the others. 
![image](https://user-images.githubusercontent.com/10436072/160276429-55863b2c-18a0-4f55-881d-74f5f4444112.png)

###### Debug info
```
AnkiDroid Version = 2.16alpha53

Android Version = 9

Manufacturer = OPPO

Model = CPH1983

Hardware = qcom

Webview User Agent = Mozilla/5.0 (Linux; Android 9; CPH1983 Build/PKQ1.190519.001; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/99.0.4844.58 Mobile Safari/537.36

ACRA UUID = 057cd16c-e7f1-4e0c-a259-8401e998e532

Scheduler = std2

Crash Reports Enabled = true

DatabaseV2 Enabled = true
```

###### Research
*Enter an [x] character to confirm the points below:*

- [x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum
- [x] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by Prince-kushwaha** [comment]
can i work on this

**Comment by mikehardy** [comment]
@Prince-kushwaha yes please! @snowtimeglass I think just updated to the new alphas (or just started really using them) and is doing a great service by checking some of the work that has built up in the alphas but not been released. Would be great to fix these issues prior to the beta cycle, and I see you've already put a PR in for one! 🏆

**Comment by snowtimeglass** [comment]
@mikehardy, thanks, I really appreciate your hard and great work. I don't mean to rush you about the issues I have reported, of course. Anyway, it will be wonderful if I can be of any help by putting PRs in future. I will positively consider challenging it again :-)
